package.json

```diff
@@ -21,16 +22,18 @@
   },
   "homepage": "https://github.com/Akinmyde/library-backend#readme",
   "devDependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.3.4",
+    "@babel/node": "^7.2.2",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.3.4",
     "babel-register": "^6.26.0",
```

Comment:
Nice job on the Babel update. Are we still using `babel-register`? If not please uninstall/remove from package.json

Reply:
we will be needing babel-register for testing with mocha